FWCore/ParameterSet/python/Config.py

```diff
@@ -2379,6 +2378,8 @@ def testTask(self):
             # Also note that the mutating visitor replaces sequences and tasks that have
             # modified contents with their modified contents, it does not modify the sequence
             # or task itself.
+            print(process.path21.dumpPython(PrintOptions()))
+            print('cms.Path(process.mproducer10+process.mproducer8+process.mproducer8+(process.mproducer8+process.mproducer9)+process.sequence3, cms.Task(), cms.Task(process.None, process.mproducer10), cms.Task(process.d, process.mesproducer, process.messource, process.mfilter, process.mproducer10, process.mproducer2, process.myTask6), process.myTask100, process.myTask5)\n')
```

Comment:
I'd rather not have the test logs have print statements.

Reply:
ah - that's a mistake!